haiku.md

```diff
@@ -0,0 +1,14 @@
+Hello World!
```

Comment:
Nice Hello world 
But it can be better

Reply:
Thanks man but there is really no time for that